selenium_tests/basic_test.py

```diff
@@ -143,7 +143,7 @@ def test_query_string_preserved(self):
         """The querystring should not be affected"""
         self.learner_setup()
         self.get("{}/learners/?q=xyz".format(self.live_server_url))
-        self.wait().until(lambda driver: self.num_elements_on_page('.learner-result', driver=driver) == 0)
+        self.wait().until(lambda driver: driver.find_element_by_class_name('learner-result'))
```

Comment:
We shouldn't change this. It asserts that there are no `learner-result` divs on the page, which would be true if the filter for `xyz` was still present on the page